zeapp/desktop/src/main/kotlin/de/berlindroid/zekompanion/desktop/ZeBadgeKompanion.kt

```diff
@@ -71,14 +72,55 @@ private fun sendResult(result: Result<Int>) {
     }
 }
 
+private fun State.EditImage.toBufferedImage(): BufferedImage {
+    val img = BufferedImage(296, 128, BufferedImage.TYPE_INT_RGB)
+    val g = img.createGraphics()
+
+    val badge = ComposePanel()
+    badge.size = Dimension(296 * 2, 128 * 2)
+    badge.setContent {
+        ImageEditorBadge(
+            image,
+            File(configFileName).readText().fromJSONtoConfig(),
+        )
+    }
+
+    return with(JFrame("badge")) {
+        layout = BorderLayout()
+        defaultCloseOperation = JFrame.DISPOSE_ON_CLOSE
+        size = Dimension(296, 128)
```

Comment:
Magic numbers 🪄

Reply:
Good point. Let me think about that for a second: Those values are the magical size of ZeBadge that will never ever change. Unless it does and then we got a problem. Maybe `/zebadge`-lib can contain the size of that badge and we reference them here?

I'll be prototyping a solution and come back with one soon :tm:.